### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by jakobloekke** [stubailo/meteor-rest#18]
I get these warnings in the terminal when using json-routes:
```
connect.multipart() will be removed in connect 3.0
visit https://github.com/senchalabs/connect/wiki/Connect-3.0 for alternatives
connect.limit() will be removed in connect 3.0
```

Everything works fine, though.

**Comment by greenik** [comment]
same here

**Comment by stubailo** [comment]
Yep, everyone will have these. I'll look into what it would take to get rid of them, but rest assured that it will not affect how your app works.

**Comment by MichaelHirn** [comment]
I am running `Meteor 1.1.0.2`, with no other packages, than the usual ones and `simple:rest` and the warnings turned to errors, which seems to corrupt the startup of the app.

```
W20150709-09:41:11.225(2)? (STDERR) connect.multipart() will be removed in connect 3.0
W20150709-09:41:11.226(2)? (STDERR) visit https://github.com/senchalabs/connect/wiki/Connect-3.0 for alternatives
W20150709-09:41:11.227(2)? (STDERR) connect.limit() will be removed in connect 3.0
```

**Comment by nooitaf** [comment]
As mentioned here https://github.com/senchalabs/connect/wiki/Connect-3.0 

replacing:
```javascript
// json-routes.js:7
WebApp.rawConnectHandlers.use(connect.bodyParser());
```
with
```javascript
WebApp.rawConnectHandlers.use(connect.urlencoded());
WebApp.rawConnectHandlers.use(connect.json());
```
"seems" to work with `json-routes` and removes the error

**Comment by stubailo** [comment]
I'd gladly accept a PR for this!

**Comment by stubailo** [comment]
Fixed, published in JsonRoutes 1.0.4

**Comment by nooitaf** [comment]
\o/ glad to help :)
